libclc/utils/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Construct LLVM version define
+set( LLVM_VERSION_DEFINE "-DHAVE_LLVM=0x${LLVM_VERSION_MAJOR}0${LLVM_VERSION_MINOR}" )
+
+# Setup prepare_builtins tools
+set(LLVM_LINK_COMPONENTS
```

Comment:
add a space after `(` to align with the style in libclc?

Reply:
Good idea, thanks. Done